sklearn/ensemble/forest.py

```diff
@@ -959,6 +959,9 @@ class RandomForestRegressor(ForestRegressor):
         reduction as feature selection criterion, and "mae" for the mean
         absolute error.
 
+    .. versionadded:: 0.18
```

Comment:
ugh i messed up this spacing (need 4 spaces), do you want to push a fix to master or should i open a new spurious pr @amueller